reference/sql_workarounds.rst

```diff
@@ -127,7 +127,7 @@ Attempting to execute a JOIN between a local and a distributed table causes an e
 
   ERROR: cannot plan queries that include both regular and partitioned relations
 
-There is a workaround: you can replicate the local table to a single shard on every worker and push the join query down to the workers. We do this by defining the table as a 'reference' table using a different table creation API. Suppose we want to join tables *here* and *there*, where *there* is already distributed but *here* is on the master database.
+There is a workaround: you can replicate the local table to a single shard on every worker and push the join query down to the workers. We do this by defining the table as a 'reference' table using a different table creation API. Suppose we want to join tables *here* and *there*, where *there* is already distributed but *here* is on the coordinator database.
```

Comment:
Any proposals for new names? A and B; local_table and distributed_table? In the absence of a theme for this section, the names "here" and "there" seemed to work although they are a bit quirky.